packages/kolibri-tools/lib/apiSpecExportTools.js

```diff
@@ -103,6 +103,7 @@ const distSpecFilePath = path.resolve(__dirname, '../dist/apiSpec.json');
 try {
   apiSpec = specModule(specFilePath);
 } catch (e) {
+  console.error(e);
```

Comment:
Logging this error helped clarify what was going wrong when confronted with an error about missing `apiSpec.json`